src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/breadcrumb.jsx

```diff
@@ -56,9 +57,9 @@ class Breadcrumb extends React.Component {
             <span className="icon" />
           </span>
           {defined(crumb.timestamp) ? (
-            <span className="dt" title={moment(crumb.timestamp).format()}>
-              {moment(crumb.timestamp).format('HH:mm:ss')}
-            </span>
+            <Tooltip title={moment(crumb.timestamp).format()}>
```

Comment:
Should we do a localized format? 
![image](https://user-images.githubusercontent.com/79684/57951822-1e4d6580-78a0-11e9-8ddc-b587d0cd47c5.png)

Reply:
`llll` could be a good one.